frontend/apps/picasso/apollo/hooks/useMarketCap.ts

```diff
@@ -13,17 +13,21 @@ export const useMarketCap = () => {
   const [picaPrice, setPicaPrice] = useState<BigNumber>(new BigNumber(0));
   const { parachainApi } = usePicassoProvider();
   useEffect(() => {
-    callbackGate((api, picassoId) => {
-      api.query.oracle.prices(
-        picassoId.toString(),
-        (result: ComposableTraitsOraclePrice) => {
-          if (!result.isEmpty) {
-            const { price, block } = result.toJSON() as any;
-            setPicaPrice(fromChainIdUnit(unwrapNumberOrHex(price)));
+    callbackGate(
+      (api, picassoId) => {
+        api.query.oracle.prices(
+          picassoId.toString(),
+          (result: ComposableTraitsOraclePrice) => {
+            if (!result.isEmpty) {
+              const { price } = result.toJSON() as any;
```

Comment:
`result.price.toString()` can be used here iirc.

Reply:
it will also need decimal conversion so please add a comment or a reminder.

Reply:
Why do we need decimal conversion here?

Reply:
@easteregg yeah that's actually ambiguous uptil now but price might also be provided by oracle as 10^12 c.c @vimukthi-git